src/screens/HomeScreen.tsx

```diff
@@ -54,3 +59,28 @@ const styles = StyleSheet.create({
     flex: 1,
   },
 });
+
+function TrackTileWrapper() {
```

Comment:
Nit: I think you could move this inside HomeScreen since it is already calling the `useAppConfig` hook. Something like:

```
const navigation = useNavigation<HomeScreenNavigation>();
const trackTile = appConfig?.homeTab?.tiles?.includes?.('trackTile') ?  <TrackTile
       onOpenTracker={(tracker, valuesContext) =>
         navigation.navigate('tiles/TrackTile', {
           tracker,
           valuesContext,
         })
       }
       title={appConfig?.homeTab?.trackTileSettings?.title}
     /> : null;
```
and then use it in TilesList:
```
          <TilesList
             TrackTile={trackTile}
             tiles={appConfig?.homeTab?.appTiles}
           />
```

Reply:
What might be nice (for a subsequent PR) is to shift all the messiness into TilesList, whereas right now both HomeScreen and TilesList feel a bit awkward. For example, maybe refactor `TilesList` to take in all of `appConfig?.homeTab` and destructure the bits it needs (e.g. `{ appTiles, tiles, ... }` etc.).  In my mind, this would leave `HomeScreen` purely showing `TilesList` and `TilesList` containing all the temporary messiness we can refactor over time.  We can eventually have TilesList host a cleaner factory for different tile types, add tile-ordering, etc. etc.  If HomeScreen is reasoning about tile types, refactoring for cleanliness will be more difficult.

Reply:
I normally don't like defining a component inside another component, but in this case, we are really just configuring it and it does make sense 👍